vizro-core/src/vizro/models/_components/form/_text_area.py

```diff
@@ -0,0 +1,49 @@
+from typing import List, Literal
+
+import dash_bootstrap_components as dbc
+from dash import html
+
+try:
+    from pydantic.v1 import Field
+except ImportError:  # pragma: no cov
+    from pydantic import Field
+
+from vizro.models import Action, VizroBaseModel
+from vizro.models._action._actions_chain import _action_validator_factory
+from vizro.models._models_utils import _log_call
+
+
+class TextArea(VizroBaseModel):
+    """Component provided to `Form` to allow user multi-line text input.
+
+    Args:
+        type (Literal["text_area"]): Defaults to `"text_area"`.
+        title (str): Title to be displayed. Defaults to `""`.
+        placeholder (str): Default text to display in input field. Defaults to `""`.
+        actions (Optional[List[Action]]): Defaults to `[]`.
+    """
+
+    type: Literal["text_area"] = "text_area"
+    title: str = Field("", description="Title to be displayed")
+    placeholder: str = Field("", description="Default text to display in input field")
+    actions: List[Action] = []
+
+    # Re-used validators
+    _set_actions = _action_validator_factory("value")
+
+    @_log_call
+    def build(self):
+        return html.Div(
+            [
+                html.Label(self.title, htmlFor=self.id) if self.title else None,
```

Comment:
I guess we would make this `dbc.Label` in future if we're going all in on bootstrap. Judging by the docs it would make a very small difference in practice compared to `html.Label` though.

Reply:
Yes, agree 👍  Would probably leave it for now, so it's consistent with the other components and then we change everything in one go.